protostar/cli/activity_indicator.py

```diff
@@ -36,3 +40,7 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_value, exc_tb):
         self.stop()
+
+
+def is_terminal() -> bool:
+    return StreamWrapper(sys.stdout, sys.stdin).isatty()
```

Comment:
duplicates code from binary_entrypoint. is this desired?

Reply:
It is. I tried to avoid creating cyclic dependencies and preserve responsiveness of the initial activity indicator.

Reply:
👍🏻 would be nice to document this but this is mergeable